src/SvgIcon/SvgIcon.js

```diff
@@ -56,6 +57,10 @@ class SvgIcon extends Component {
     hovered: false,
   };
 
+  shouldComponentUpdate = (nextProps, nextState) => {
```

Comment:
I don't think that we should implement a `shouldComponentUpdate` for any component that accepts a `children` property. The check will almost fail unless users are caching their children manually or with `babel-plugin-transform-react-constant-elements`.